build.cmd

```diff
@@ -674,7 +674,7 @@ if %__BuildCoreLib% EQU 1 (
         set IbcMergePath=%__PackagesDir%\microsoft.dotnet.ibcmerge\!__IbcMergeVersion!\lib\net45\ibcmerge.exe
         if exist !IbcMergePath! (
             echo %__MsgPrefix%Optimizing using IBC training data
-            set OptimizationDataDir=%__PackagesDir%\optimization.%__BuildOS%-%__BuildArch%.IBC.CoreCLR\!__IbcOptDataVersion!\data\System.Private.CoreLib\
+            set OptimizationDataDir=%__PackagesDir%\optimization.%__BuildOS%-%__BuildArch%.IBC.CoreCLR\!__IbcOptDataVersion!\data\System.Private.CoreLib.dll\
```

Comment:
I'm looking at `optimization.windows_nt-x64.IBC.CoreCLR 99.99.99-master-20190306.1`, the path I see is still without the DLL extension. My guess is we'll have to wait until we get a package now that IBC is enabled for official builds. If all the legs pass, I expect `99.99.99-master-20190306.3` to have the new format. We'd just have to either wait for that to get in through dependency flow, or get the version bump done in this PR to prevent

Reply:
Yes, I will wait to check this in until I see the dependency flow update.